doc/requirements-doc.txt

```diff
@@ -13,6 +13,7 @@ pickle5
 pillow
 pydantic<1.7
 pygments
+pytorch-lightning
```

Comment:
can we avoid making pytorch lightning a requirement of the documentation? we have tried to keep this build as minimal as possible

Reply:
yes I removed it. It's not even needed.